Tasks/JavaToolInstaller/task.json

```diff
@@ -11,116 +11,116 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 0,
-        "Minor": 126,
-        "Patch": 1
+        "Major": 1,
+        "Minor": 127,
+        "Patch": 0
     },
     "preview": true,
```

Comment:
If we are setting major version to 1, it makes sense to remove the "preview" flag.